### PR TITLE
Remove dor-services-client.

### PR DESCRIPTION
**Pull request by justinlittman**
## Why was this change made? 🤔
To make it easier to test cocina models changes without cutting a new release.

## How was this change tested? 🤨

⚡ ⚠ If this change has cross service impact, including data writes to shared file systems, ***run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test in [stage|qa] environment, in addition to specs. ⚡

local, sdr-deploy

### CODE REVIEW

File: lib/fedora_cache.rb [hunk 1]
```diff
@@ -157,6 +157,6 @@ def apo_for(rels_ext)
   end
 
   def fetch_tags(druid)
-    Dor::Services::Client.object(druid).administrative_tags.list
+    AdministrativeTags.for(identifier: druid)
```
Comment (mjgiarlo) [inline lib/fedora_cache.rb:160]: Hmm, I'm curious why you use the service class here and use the HTTP API in `lib/fedora_leader.rb`

File: Gemfile [hunk 2]
```diff
@@ -46,7 +46,6 @@ gem 'uuidtools', '~> 2.1.4'
 gem 'whenever', require: false
 
 group :development do
-  gem 'dor-services-client' # used by lib/fedora_cache.rb and lib/fedora_loader.rb
```
Comment (ndushay) [inline Gemfile:49]: I was always a bit uncomfy with the app requiring its own client

File: lib/fedora_cache.rb (hunk 1 above)
Comment (justinlittman) [reply lib/fedora_cache.rb:160]: This invokes the implementation in `lib/fedora_loader.rb` (which is a monkeypatch of `AdministrativeTags`).

**Review by ndushay — APPROVED** [review]
I'm good with this as long as it was tested on the bin/*** scripts.

File: Gemfile (hunk 2 above)
Comment (justinlittman) [reply Gemfile:49]: It was a cheat.

File: lib/fedora_cache.rb (hunk 1 above)
Comment (mjgiarlo) [reply lib/fedora_cache.rb:160]: Aha! Right, now I see. Perfect.

**Pull request merged**